modules/apps/product-navigation/product-navigation-product-menu-api/src/test/java/com/liferay/product/navigation/product/menu/display/context/ProductMenuDisplayContextTest.java

```diff
@@ -0,0 +1,238 @@
+
+import org.mockito.ArgumentMatchers;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+
+import org.springframework.mock.web.MockHttpServletRequest;
+
+/**
+ * @author Eudaldo Alonso
+ */
+public class ProductMenuDisplayContextTest {
+
+	@ClassRule
+	@Rule
+	public static final LiferayUnitTestRule liferayUnitTestRule =
+		LiferayUnitTestRule.INSTANCE;
+
+	@Before
+	public void setUp() throws Exception {
+		_themeDisplay = Mockito.mock(ThemeDisplay.class);
+
+		_mockHttpServletRequest = new MockHttpServletRequest();
+
+		_mockPortletRequest = new MockLiferayResourceRequest();
+
+		_mockPortletRequest.setAttribute(
+			PortletServlet.PORTLET_SERVLET_REQUEST, _mockHttpServletRequest);
+
+		_mockPortletRequest.setAttribute(WebKeys.THEME_DISPLAY, _themeDisplay);
+
+		_mockHttpServletRequest.setAttribute(
+			JavaConstants.JAVAX_PORTLET_REQUEST, _mockPortletRequest);
+
+		_mockHttpServletRequest.setAttribute(
+			WebKeys.THEME_DISPLAY, _themeDisplay);
+
+		_applicationsMenuInstanceConfiguration = Mockito.mock(
+			ApplicationsMenuInstanceConfiguration.class);
+
+		_panelCategoryHelper = Mockito.mock(PanelCategoryHelper.class);
+
+		_mockPortletRequest.setAttribute(
+			ApplicationListWebKeys.PANEL_CATEGORY_HELPER, _panelCategoryHelper);
+
+		PanelCategory panel1 = _newPanelCategory("panel1");
+		PanelCategory panel2 = _newPanelCategory("panel2");
+		PanelCategory panel3 = _newPanelCategory("panel3");
+
+		_panelCategories = Arrays.asList(panel1, panel2, panel3);
+	}
+
+	@Test
+	public void testShouldReturnOnlyPanelCategoriesContainsPanelApps() {
+		try (MockedStatic<PortalUtil> portalUtilMockedStatic =
+				Mockito.mockStatic(PortalUtil.class);
+			MockedStatic<ConfigurationProviderUtil>
+				configurationProviderUtilMockedStatic = Mockito.mockStatic(
+					ConfigurationProviderUtil.class)) {
+
+			portalUtilMockedStatic.when(
+				() -> PortalUtil.getHttpServletRequest(ArgumentMatchers.any())
+			).thenReturn(
+				_mockHttpServletRequest
+			);
+
+			configurationProviderUtilMockedStatic.when(
+				() -> ConfigurationProviderUtil.getCompanyConfiguration(
+					ArgumentMatchers.any(), ArgumentMatchers.anyLong())
+			).thenReturn(
+				_applicationsMenuInstanceConfiguration
+			);
+
+			Mockito.when(
+				_applicationsMenuInstanceConfiguration.enableApplicationsMenu()
+			).thenReturn(
+				false
+			);
+
+			Mockito.when(
+				_panelCategoryHelper.getPanelCategories(
+					PanelCategoryKeys.APPLICATIONS_MENU, _themeDisplay)
+			).thenReturn(
+				_panelCategories
+			);
+
+			Mockito.when(
+				_panelCategoryHelper.getAllPanelApps(
+					ArgumentMatchers.anyString())
+			).thenReturn(
+				Arrays.asList(Mockito.mock(PanelApp.class))
+			);
+
+			Mockito.when(
+				_panelCategoryHelper.getAllPanelApps("panel2")
+			).thenReturn(
+				Collections.emptyList()
+			);
+
+			ProductMenuDisplayContext context = new ProductMenuDisplayContext(
+				_mockPortletRequest);
+
+			List<PanelCategory> result = context.getChildPanelCategories();
```

Comment:
When the method is a get, the variable will be 95% of the time called whatever is after the `get`.

```
List<PanelCategory> childPanelCategories = context.getChildPanelCategories();
```

Other cases are when we have multiple variables of the same time and we want to differentiate. For that we can name them:

```
Type type1 = new Type();
Type type2 = new Type();
Type type3 = getType();
```
 or use different names 
```
List<Type> userTypes = getUserTypes();
List<Type> systemTypes = getSystemTypes();
```